lldb/test/API/commands/frame/recognizer/TestFrameRecognizer.py

```diff
@@ -305,7 +307,9 @@ def test_frame_recognizer_target_specific(self):
 
         self.expect(
             "frame recognizer list",
-            substrs=["recognizer.MyFrameRecognizer, module a.out, symbol bar"],
+            substrs=[
+                "recognizer.MyFrameRecognizer, module a.out, demangled symbol bar"
+            ],
```

Comment:
Unfortunately, I was not able to write such a test case. Note that the `frame recognizer add` command does not yet expose a way to match on anything else than demangled names. I agree that we should be adding test cases here, as soon as `frame recognizer add` supports choosing the symbol mangling.

@adrian-prantl Are you planning to extend `frame recognizer add` accordingly?

Reply:
> Unfortunately, I was not able to write such a test case. Note that the `frame recognizer add` command does not yet expose a way to match on anything else than demangled names. I agree that we should be adding test cases here, as soon as `frame recognizer add` supports choosing the symbol mangling.
> 
> @adrian-prantl Are you planning to extend `frame recognizer add` accordingly?

It's not on my immediate todo list, but it would be an obvious next step.